docs/deployment/migration-guide.md

```diff
@@ -24,7 +24,7 @@
 * Since Kyuubi 1.7, Kyuubi returns engine's information for `GetInfo` request instead of server. To restore the previous behavior, set `kyuubi.server.info.provider` to `SERVER`.
 * Since Kyuubi 1.7, Kyuubi session type `SQL` is refactored to `INTERACTIVE`, because Kyuubi supports not only `SQL` session, but also `SCALA` and `PYTHON` sessions.
   User need to use `INTERACTIVE` sessionType to look up the session event.
-* Since Kyuubi 1.7, the REST API of `Open(create) a session` will not contains parameters `user` `password` and `IpAddr`. User and password should be set in `Authorization` of http request if needed.
+* Since Kyuubi 1.7, the REST API of `Open(create) a session` will not contain parameters `protocolVersion`, `user` `password` and `IpAddr`. User and password should be set in `Authorization` of http request if needed.
```

Comment:
question: what will happen if a user uses Kyuubi 1.7.0 client access Kyuubi Server 1.7.1

Reply:
Nothing happen. By default，protocol is 0， V1

Reply:
The unknown field should be ignore.

Reply:
We need doc it separately with patch-numbered releases, 1.7 looks aimless